### PR TITLE
Adding litellm to relevant resources

### PR DESCRIPTION
**Pull request by krrishdholakia**
Adding litellm to relevant resources

### CODE REVIEW

**Comment by krrishdholakia** [comment]
cc: @simonpfish

**Review by simonpfish — CHANGES_REQUESTED** [review]
Nice, thanks for contributing to our related resources!

I think it's important that you phrase this description here around the actual utility that this library gives to developers, and not just the fact that it can call many LLM APIs. Langchain, Haystack, LLamaIndex can all already do that. What differentiates LiteLLM?

I would suggest perhaps also updating the phrasing in your repo's description to be more inclusive and not about replacing OpenAI. I like the framing in your README slightly better "Call all LLM APIs using the OpenAI format"

CleanShot 2023-10-03 at 18 07 34@2x

File: related_resources.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ People are writing great tools and papers for improving outputs from GPT. Here a
 - [Guidance](https://github.com/microsoft/guidance): A handy looking Python library from Microsoft that uses Handlebars templating to interleave generation, prompting, and logical control.
 - [Haystack](https://github.com/deepset-ai/haystack): Open-source LLM orchestration framework to build customizable, production-ready LLM applications in Python.
 - [LangChain](https://github.com/hwchase17/langchain): A popular Python/JavaScript library for chaining sequences of language model prompts.
+- [🚅 LiteLLM](https://github.com/BerriAI/litellm): Easily call Anthropic/Huggingface/etc. using the OpenAI format
```
[the remark this answers is not in the document]
Comment (krrishdholakia) [reply related_resources.md:14]: is that a requirement? This is a pretty consistent part of our branding, and I'd prefer to keep it.

**Comment by krrishdholakia** [comment]
Update made @simonpfish 
Screenshot 2023-10-03 at 6 17 34 PM

**Comment by krrishdholakia** [comment]
Bump on this @simonpfish

File: related_resources.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (simonpfish) [reply related_resources.md:14]: Yes that's a requirement. We don't want to give preferential treatment to any library, and emojis stand out.

**Comment by simonpfish** [comment]
I'm waiting for you to answer my previous question:

> I think it's important that you phrase this description here around the actual utility that this library gives to developers, and not just the fact that it can call many LLM APIs. Langchain, Haystack, LLamaIndex can all already do that. What differentiates LiteLLM?

**Comment by krrishdholakia** [comment]
I thought i did that by updating the description based on the one you recommended - 
Screenshot 2023-10-04 at 11 33 20 AM
Screenshot 2023-10-04 at 11 33 29 AM

is there another place where the change was required?

File: related_resources.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (krrishdholakia) [reply related_resources.md:14]: ok - updated.

**Comment by krrishdholakia** [comment]
@simonpfish LiteLLM doesn't compete with Langchain, LlamaIndex etc. We actually have integrations with both of them: 
- https://python.langchain.com/docs/integrations/chat/litellm
- https://docs.llamaindex.ai/en/latest/examples/llm/litellm.html

These were because of community requests - e.g.: https://github.com/run-llama/llama_index/issues/7458

We built this library because nothing solves the consistent model I/O problem that exists when you go cross-provider. 

LiteLLM makes it easy for you to call different LLM APIs with a consistent input-output format (i.e. the OpenAI format). 

Please let me know what is unclear here.

File: related_resources.md [hunk 2]
```diff
@@ -11,6 +11,7 @@ People are writing great tools and papers for improving outputs from GPT. Here a
 - [Guidance](https://github.com/microsoft/guidance): A handy looking Python library from Microsoft that uses Handlebars templating to interleave generation, prompting, and logical control.
 - [Haystack](https://github.com/deepset-ai/haystack): Open-source LLM orchestration framework to build customizable, production-ready LLM applications in Python.
 - [LangChain](https://github.com/hwchase17/langchain): A popular Python/JavaScript library for chaining sequences of language model prompts.
+- [LiteLLM](https://github.com/BerriAI/litellm): Easily call Anthropic/Huggingface/etc. using the OpenAI format
```
Comment (simonpfish) [inline related_resources.md:14]:
```suggestion
- [LiteLLM](https://github.com/BerriAI/litellm): Minimal Python library that allows you to call various LLM APIs with a consistent format. 
```

**Comment by simonpfish** [comment]
Got it. Just suggested some updates to the description to better capture this.

**Comment by krrishdholakia** [comment]
Updated based on feedback, are there any remaining blockers for merge @simonpfish

**Comment by simonpfish** [comment]
Rather than using an imperative sentence, let's use the same descriptive format as all the other resources listed. For example, "A Python library that allows you to..."

**Comment by krrishdholakia** [comment]
Done @simonpfish. Any remaining blockers for merge

**Comment by krrishdholakia** [comment]
bump on this @simonpfish

**Comment by simonpfish** [comment]
No need for all the bumps. This is on my queue and I'll get to it as soon as I can. We're all busy people, no need to spam our inboxes.

File: related_resources.md [hunk 3]
```diff
@@ -11,6 +11,7 @@ People are writing great tools and papers for improving outputs from GPT. Here a
 - [Guidance](https://github.com/microsoft/guidance): A handy looking Python library from Microsoft that uses Handlebars templating to interleave generation, prompting, and logical control.
 - [Haystack](https://github.com/deepset-ai/haystack): Open-source LLM orchestration framework to build customizable, production-ready LLM applications in Python.
 - [LangChain](https://github.com/hwchase17/langchain): A popular Python/JavaScript library for chaining sequences of language model prompts.
+- [LiteLLM](https://github.com/BerriAI/litellm): A Python library to call various LLM APIs (OpenAI/Azure/Huggingface/etc.) using the OpenAI format
```
Comment (simonpfish) [inline related_resources.md:14]:
```suggestion
- [LiteLLM](https://github.com/BerriAI/litellm): A minimal Python library for calling LLM APIs with a consistent format. 
```

**Comment by simonpfish** [comment]
Added a period and modified the wording to be more descriptive and general.

**Comment by krrishdholakia** [comment]
Thanks @simonpfish 

I don't see it yet. Is there a delay before it's live? 
Screenshot 2023-10-05 at 11 26 46 AM

**Pull request merged**